modules/scanning-delegate-role/main.tf

```diff
@@ -114,6 +115,49 @@ data "aws_iam_policy_document" "scanning_orchestrator_policy_document" {
     }
   }
 
+  statement {
+    sid    = "DatadogAgentlessScannerCopyImage"
+    effect = "Allow"
+    actions = [
+      "ec2:CopyImage"
```

Comment:
`CopyImage` also requires a permission on the resource `snapshot*` resources. See [here](https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazonec2.html#amazonec2-CopyImage).